modules/op-running-pipeline-run-and-task-run-with-custom-scc-and-service-account.adoc

```diff
@@ -1,8 +1,11 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/configuring-security-context-for-pods.adoc
 :_content-type: PROCEDURE
 [id="op-running-pipeline-run-and-task-run-with-custom-scc-and-service-account_{context}"]
 = Running pipeline run and task run by using a custom SCC and a custom service account
 
-When using the `pipelines-scc` security context constraint (SCC) associated with the default `pipelines` service account, the pipeline run and task run pods may face timeouts. This happens because in the default `pipelines-scc` SCC, the `fsGroup.type` parameter is set to `MustRunAs`. 
+When using the `pipelines-scc` security context constraint (SCC) associated with the default `pipelines` service account, the pipeline run and task run pods may face timeouts. This happens because in the default `pipelines-scc` SCC, the `fsGroup.type` parameter is set to `MustRunAs`.
```

Comment:
```suggestion
When using the `pipelines-scc` security context constraint (SCC) associated with the default `pipelines` service account, the pipeline run and task run pods might face timeouts. This happens because in the default `pipelines-scc` SCC, the `fsGroup.type` parameter is set to `MustRunAs`.
```
IBM Style Guide suggests using "might" instead of "may" in cases like this. https://www.ibm.com/docs/en/ibm-style?topic=word-usage#m